src/main/java/com/github/admissionCommittee/model/User.java

```diff
@@ -0,0 +1,130 @@
+package com.github.admissionCommittee.model;
+
+import javax.persistence.*;
+
+import java.util.Set;
+
+@Entity
+@Table(name = "users")
+@AttributeOverride(name = "id", column = @Column(name = "user_id",
+        nullable = false))
+public class User extends NamedEntity {
+
+    @Column(name = "role")
+    private Role role;
+    @Column(name = "name")
+    private String name;
+    @Column(name = "patronymic")
+    private String patronymic;
+    @Column(name = "surname")
+    private String surname;
+    @Column(name = "passport")
+    private String passport;
+    @Column(name = "email")
+    private String email;
+    @Column(name = "phone")
+    private String phone;
+    @OneToMany(mappedBy = "user")
+    @Column(name = "faculty")
+    private Set<Faculty> faculty;
+    @OneToOne(fetch = FetchType.EAGER)

```

Comment:
EAGER is actually bad... it's better to use LAZY + Transactional when you need it